src/ansys/dpf/core/operators/geo/normals_provider_nl.py

```diff
@@ -12,13 +12,13 @@
 
 
 class normals_provider_nl(Operator):
-    """Computes the normals on nodes/elements based on integration points
-    (more accurate for non-linear elements) on a skin mesh.
+    """Computes the normals on nodes/faces/elements based on integration
+    points (more accurate for non-linear elements) on a skin mesh.
 
     Parameters
     ----------
     mesh : MeshedRegion
-        Skin or shell mesh region.
+        Skin, face, or shell mesh region.
     mesh_scoping : Scoping, optional
         Elemental, elementalnodal, or nodal scoping.
         location derived from this.
```

Comment:
```suggestion
        Mesh scoping, which can be elemental, elemental nodal, or nodal.
        Location is derived from this.
```
I'm not sure if my suggestion here is correct, but line 24 was an incomplete sentence.